org.gnome.gitlab.sophie_h.PikaBackup.yml

```diff
@@ -0,0 +1,82 @@
+---
+app-id: org.gnome.gitlab.sophie_h.PikaBackup
+command: pika-backup
+runtime: org.gnome.Platform
+runtime-version: "3.36"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+
+finish-args:
+  - --filesystem=host
+  - --share=network
+  - --socket=wayland
+  # X11
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
you need one of socket=x11 or fallback-x11, not both. fallback-x11 is preferred and means your app will use wayland by default in a wayland environment.

Reply:
Left x11 in to keep compatibility with debian old-stable (flatpak < 1.0), but probably I'm missing a million required features there anyways. Removed it.

Reply:
And thanks for the feedback!

Reply:
< Flatpak 1.2 is about 20% of Flathub downloads fwiw https://ahayzen.com/direct/flathub.html but yeah, that's fair! If you absolutely know you need a feature that isn't present in an older version you can use --require-version like so https://github.com/flathub/com.visualstudio.code/blob/master/com.visualstudio.code.yaml#L11